lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -612,8 +612,11 @@
     // ====================
 
     // UI: message
-    .message {
+    .wrapper-message-primary {
```

Comment:
Because this class is only used to implement clearfix, should we have a more general clearfix class that is used instead? @talbs thoughts about these sort of util classes? I just don't want to have multiple classes that are doing the same thing but with abstracted names.